stable/artifactory/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: artifactory
 home: https://www.jfrog.com/artifactory/
-version: 6.2.5
+version: 6.3.1
```

Comment:
Because the API to the chart change (e.g., values.yaml file structure) in a backwards breaking way this should be 7.0.0 instead of a minor bump. That is to follow [semver](http://semver.org) rules.